server/db.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	_ "github.com/lib/pq"
+	"golang.org/x/crypto/bcrypt"
+)
+
+// LoginUser checks bcrypt hashed passwords match in the users table
+func LoginUser(u User) bool {
+	db := GetDB()
```

Comment:
Yeah, lets defer(no pun intended) fixing this for now. It will work fine for the prototype